pkg/inject/envoy.go

```diff
@@ -274,3 +277,11 @@ func (m *envoyMutator) getUseDualStackEndpoint(useDualStackEndpoint bool) string
 		return "0"
 	}
 }
+
+func (m *envoyMutator) getUseFipsEndpoint(useFipsEndpoint bool) string {
+	if useFipsEndpoint {
+		return "1"
```

Comment:
why we pass String instead of bool here?

Reply:
Actually Controller takes boolean and it converts into String because Envoy accepts FIPS endpoint as ENV variable which is String. so we are converting and passing it to envoy. If I am not wrong.

Reply:
Makes sense, thanks